service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsControllerTest.java

```diff
@@ -131,6 +157,15 @@ private void assertC2BundleDocument(C2DocumentBundle documentBundle, String desc
         assertThat(documentBundle.getDescription()).isEqualTo(description);
     }
 
+    private CallbackRequest createCallbackRequestWithTempC2BundleWithoutDocument() {
```

Comment:
Not a fan of these `createCallbackRequestWithTempC2BundleWithoutDocument` & `createCallbackRequestWithTempC2Bundle`. Could we not have one method `createCallBackRequest` which takes in data as a param?

Reply:
done with your help :)